jxl/Cargo.toml

```diff
@@ -26,6 +26,7 @@ paste = "1.0.15"
 rand = "0.8.5"
 rand_xorshift = "0.3.0"
 test-log = { version = "0.2.16", features = ["trace"] }
+jxl_macros = { path = "../jxl_macros", version = "=0.1.0", features = ["test"] }
```

Comment:
I think you can remove path and set version="*" here, can you try?

Reply:
I could set version = "*", but I couldn't remove path.

```
error: failed to load manifest for workspace member `/usr/local/google/home/zond/projects/jxl-rs/jxl`
referenced by workspace at `/usr/local/google/home/zond/projects/jxl-rs/Cargo.toml`

Caused by:
  failed to parse manifest at `/usr/local/google/home/zond/projects/jxl-rs/jxl/Cargo.toml`

Caused by:
  Dependency 'jxl_macros' has different source paths depending on the build target. Each dependency must have a single canonical source path irrespective of build target.
```